opencsp/common/lib/cv/fiducials/AbstractFiducials.py

```diff
@@ -18,77 +18,123 @@ class AbstractFiducials(ABC):
+
+        Returns
+        -------
+        p2.Pxy
+            The origin point(s) of the fiducial.
+        """
+
+    # "ChatGPT 4o" assisted with generating this docstring.
 
     @property
     @abstractmethod
     def rotation(self) -> scipy.spatial.transform.Rotation:
         """
-        The pointing of the normal vector(s) of this instance.
-        This is relative to the camera's reference frame, where x is positive
-        to the right, y is positive down, and z is positive in (away from the
-        camera).
+        Get the pointing of the normal vector(s) of this instance.
+
+        Returns
+        -------
+        scipy.spatial.transform.Rotation
+            The rotation of the fiducial relative to the camera's reference frame.
+
+        Notes
+        -----
+        This can be used to describe the forward transformation from the camera's perspective. For example, an ArUco
+        marker whose origin is in the center of the image and is facing towards the camera could have the rotation
+        defined as:
 
-        This can be used to describe the forward transformation from the
-        camera's perspective. For example, an aruco marker whose origin is in
-        the center of the image and is facing towards the camera could have the
-        rotation::
+        .. code-block:: python
 
             Rotation.from_euler('y', np.pi)
 
-        If that same aruco marker was also placed upside down, then it's
-        rotation could be::
+        If that same ArUco marker was also placed upside down, then its rotation could be defined as:
+
+        .. code-block:: python
 
             Rotation.from_euler(
                 'yz',
-                [ [np.pi, 0],
-                  [0,     np.pi] ]
+                [[np.pi, 0],
+                [0,     np.pi]]
             )
 
-        Not that this just describes rotation, and not the translation. We call
-        the rotation and translation together the orientation.
+        Note that this just describes rotation, and not the translation. We call the rotation and translation together
+        the orientation.
         """
 
+    # "ChatGPT 4o" assisted with generating this docstring.
+
     @property
     @abstractmethod
     def size(self) -> list[float]:
-        """The scale(s) of this fiducial, in pixels, relative to its longest axis.
-        For example, if the fiducial is a square QR-code and is oriented tangent
-        to the camera, then the scale will be the number of pixels from one
-        corner to the other."""  # TODO is this a good definition?
+        """
+        Get the scale(s) of this fiducial, in pixels, relative to its longest axis.
+
+        Returns
+        -------
+        list[float]
+            The sizes of the fiducial in pixels. For example, if the fiducial is a square QR-code and is oriented tangent
+            to the camera, then the scale will be the number of pixels from one corner to the other.
```

Comment:
Maybe I just don't understand how documentation should be written...

```suggestion
        Get the scale(s) of this fiducial, in pixels, relative to its longest axis.
        
        As an example, if the fiducial is a square QR-code and is oriented tangent
        to the camera, then the scale will be the number of pixels from one
        corner to the other.

        Returns
        -------
        list[float]
            The sizes of the fiducial in pixels.
```